core/app/views/spree/shipment_mailer/shipped_email.html.erb

```diff
@@ -1,30 +1,37 @@
-<table>
+<table class="row header">
   <tr>
     <td>
       <p class="lede">
         <%= Spree.t('shipment_mailer.shipped_email.dear_customer') %>
       </p>
       <p>
-        <%= Spree.t('shipment_mailer.shipped_email.instructions') %>
+        <%= Spree.t('shipment_mailer.shipped_email.instructions', number: @shipment.order.number) %>
       </p>
       <p>
         <%= Spree.t('shipment_mailer.shipped_email.shipment_summary') %>
       </p>
-      <table>
-        <% @shipment.manifest.each do |item| %>
-          <tr>
-            <td><%= item.variant.sku %></td>
-            <td><%= item.variant.product.name %></td>
-            <td><%= item.variant.options_text %></td>
-          </tr>
-        <% end %>
+      <table class="container">
+        <tr>
+          <td class="wrapper last">
+            <table class="twelve columns">
+              <%= render collection: @shipment.manifest.map(&:line_item), partial: 'spree/order_mailer/line_item' %>
```

Comment:
@damianlegawiec Agree. But this gives an impression that this partial is not been used anywhere other than order_mailer and may break shipment mailer if edited for order_mailer.
To avoid clash with any other partial we can name it like `mailer_line_item` and can be used as `spree/shared/mailer_line_item` or `spree/admin/shared/mailer_line_item`.